src/lane_follow/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lane_follow)
+
+## Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS
+        roscpp
+        rospy
+        image_transport
+        cv_bridge
+        )
+
+find_package(OpenCV REQUIRED)
+
+find_package(sb_utils REQUIRED)
+
+add_definitions(-std=c++14)
+
+catkin_package(
+    #  INCLUDE_DIRS include
+)
+
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
+## Your package locations should be listed before other locations
+# include_directories(include)
+include_directories(
+    ${catkin_INCLUDE_DIRS}
+    ${sb_utils_INCLUDE_DIRS}
+    ${OpenCV_INCLUDE_DIRS}
+    ./include
+)
+
+## Declare a C++ executable
+# add_executable(sample_package_node src/sample_package_node.cpp)
+add_executable(lane_follower
+    src/line_detect.cpp    
+    src/LineDetect.cpp 
```

Comment:
This line will break CI currently (as `src/LineDetect.cpp` does not exist), but we should fix that by splitting the header into header and `.cpp` file.